components/MyApplicationView.tsx

```diff
@@ -1,65 +1,71 @@
-import { GetApplicationDataQuery } from "lib/generated/graphql"
+import { GetApplicationDataQuery } from 'lib/generated/graphql';
 
 interface MyApplicationViewProps {
-    appData: GetApplicationDataQuery["filledApplications"];
-    typeformAppData: NonNullable<GetApplicationDataQuery["me"]["profile"]>["typeformSubmissions"];
+  appData: GetApplicationDataQuery['filledApplications'];
+  typeformAppData: NonNullable<GetApplicationDataQuery['me']['profile']>['typeformSubmissions'];
 }
-
 export default function MyApplicationView({ appData, typeformAppData }: MyApplicationViewProps) {
-    return <div className="w-full flex flex-col items-center lg:flex-row flex-wrap gap-[30px]">
-        {[...appData.map(({
-            status, app: { name }
-        }) => (
-        <div key={name} className="flex flex-col items-end w-fit">
+  return appData[0] ? (
```

Comment:
Need to check if both `appData` and `typeformAppData` is empty before printing out "No application found".